UI/Web/src/app/pdf-reader/_components/pdf-reader/pdf-reader.component.ts

```diff
@@ -126,7 +126,6 @@ export class PdfReaderComponent implements OnInit, OnDestroy {
       this.navService.hideNavBar();
       this.themeService.clearThemes();
       this.navService.hideSideNav();
-      pdfDefaultOptions.disableAutoFetch = true;
```

Comment:
We need to call this out that we removed this in the PR description (changelog).